cmd/analyzer/Makefile

```diff
@@ -1,14 +1,14 @@
 version ?= $(shell git describe --tag --dirty)
 
 ldflags := -X 'main.Version=$(version)'
-builddir := ../../build/
+builddir := ../../build
 
 .PHONY: all build run
 
 all: build
 
 build:
-	go build -o $(builddir) -ldflags="$(ldflags)" analyzer.go
+	go build -o $(builddir)/analyzer -ldflags="$(ldflags)" main/main.go
```

Comment:
File with `main()` could be `main/analyzer.go` just not to need the extra flags for `go build` (would need it when running it manually as well)

Reply:
Oh good point